src/ZX/circuit_extraction.jl

```diff
@@ -285,7 +285,7 @@ function update_frontier!(zxg::ZXGraph{T, P}, gads::Set{T}, frontier::Vector{T},
         v = frontier[i]
         nb_v = neighbors(zxg, v)
         u = findfirst([u in gads for u in nb_v])
-        if u !== nothing
+          if !isnothing(u)
```

Comment:
indentation is still messup, could you please check your formatting script? did it include this directory?